e2e/grpc-options/services/Echo/echo/echo.proto

```
@@ -0,0 +1,24 @@
+syntax = "proto3";
+
+package echo;
```

Buf (1.72.0)

[error] Files with package "echo" must be within a directory "echo" relative to root but were in directory "e2e/grpc-options/services/Echo".

(PACKAGE_DIRECTORY_MATCH)